### PR TITLE
Add Dutch translation #536

### LINKED ISSUE #536: Update Dutch Translation
## Description

Update the Dutch translation to the new [locale format](https://github.com/tripleblindmarket/covid-safe-paths/tree/develop/app/locales).

## Motivation

In PR https://github.com/tripleblindmarket/covid-safe-paths/pull/310, a redo of the Dutch translations was [requested](https://github.com/tripleblindmarket/covid-safe-paths/pull/310#issuecomment-611904883). 

## Assignment

I will handle the translation, restructuring and implementation, based earlier translations done by the community.  Any updates will be posted in this issue.

### PR DESCRIPTION
**Pull request by danoszz**
Closes #536 - Dutch translation is added

### Testing

* The Dutch translation is tested on 3 different native speakers to verify accessibility and clarity
* There still could be mistakes, if you find them, please notify me and I will improve the translation

### CODE REVIEW

File: app/locales/languages.js [hunk 1]
```diff
@@ -89,6 +90,7 @@ i18next.init({
     ru: { label: 'Русский', translation: ru },
     sk: { label: 'Slovak', translation: sk },
     vi: { label: 'Vietnamese', translation: vi },
+    nl: { label: 'Nederlands', translation: nl },
```
Comment (alpita-masurkar) [inline app/locales/languages.js:93]: you'll need to move this up in the alphabetical order

File: app/locales/nl.json [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+    "settings_title": "Dashboard",
+    "share_location_data": "Deel uw locatie gegevens",
+    "show_overlap": "Bekijk publieke gegevens",
+    "team": "Team",
+    "team_para": "Ons team bestaat uit een consortium van epidemiologen, ingenieurs, data wetenschappers, digitale privacy-evangelisten, professoren en onderzoekers van gerenommeerde instellingen, waaronder: MIT, Harvard, The Mayo Clinic, TripleBlind, EyeNetra, Ernst & Young en Link Ventures.",
+    "terms_of_use": "Gebruiksvoorwaarden",
+    "terms_of_use_url": "https://docs.google.com/document/d/1mtdal_pywsKZVMXLHjjj5eKznipPLP8sM1HwFTIhjo0/edit#",
+    "test_positive_subtitle": "Uw privégegevens kunnen worden overgedragen aan gezondheidsautoriteiten, een back-up worden gemaakt of anderszins worden gedeeld.",
+    "tested_positive_title": "Deel locatie geschiedenis"
+  }
```
Comment (alpita-masurkar) [inline app/locales/nl.json:101]: there are more new keys @danoszz will you have a look at the current en.json and make sure you have them?

Comment (danoszz) [reply app/locales/nl.json:101]: Yes, got it!

**Comment by danoszz** [comment]
### Note

Here in the Netherlands, it's not possible to get tested without having severe symptoms. Also, the Mayo Clinics information could be not true here. So I've decided on the following:

* Change: "go to the website from the Mayo Clinic" -> "Go to the website of the RIVM" This is the Dutch Governmental Institute of Health
* Keep: "Latest news from Mayo Clinic." but add 'USA' so the user is aware that this news will be not from the Netherlands
* Remove: "If you have no symptoms but still would like to be tested you can go to your nearest testing site." to our national advise: "contact your health provider to asses the situation"

**Pull request merged**